webapp/src/components/FileUploader.tsx

```diff
@@ -45,7 +45,7 @@ export const FileUploader: React.FC<FileUploaderProps> = forwardRef<HTMLInputEle
                 className={classes.root}
                 accept={acceptedExtensions?.join(',')}
                 onChange={onChange}
-                title="Upload a .pdf, .txt, .jpg, .png or .tiff file"
+                title="Upload a .pdf, .txt, .jpg, .png .tiff or .csv file"
```

Comment:
nit: 
```suggestion
                title="Upload a .pdf, .txt, .jpg, .png, .tiff or .csv file"
```

Reply:
I've added the missing comma, and moved CSV next to the text-type of files